src/io/procops.c

```diff
@@ -49,29 +49,45 @@ static char * ANSIToUTF8(MVMuint16 acp, const char * str)
     return result;
 }
 
+MVM_PUBLIC char **
```

Comment:
At first I did not have `MVM_PUBLIC`, but then `link` failed. Maybe it is an issue with the ordering of filenames.